content/en/blog/2024/prometheus-compatibility-survey/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Insights from the Prometheus Compatibility Survey
+linkTitle: Prometheus Compatibility Survey
+date: 2024-06-10
+author: '[David Ashpole](https://github.com/dashpole) (Google)'
+issue: https://github.com/open-telemetry/sig-end-user/issues/24
+sig: End-User SIG
+cSpell:ignore: Ashpole
+---
+
+[Prometheus](https://prometheus.io/) and OpenTelemetry are two of the most
+active and popular projects in the
+[CNCF observability landscape](https://landscape.cncf.io/guide#observability-and-analysis--observability).
+The two communities have been working together since the early days of
+OpenTelemetry to improve the compatibility between the two projects. The
+OpenTelemetry Prometheus SIG has been leading this effort, with the active
+participation of maintainers from both OpenTelemetry and Prometheus.
+
+At this point, there is a
+[detailed, experimental specification](/docs/specs/otel/compatibility/prometheus_and_openmetrics/)
+describing how to convert between the
+[OpenTelemetry metrics data model](/docs/specs/otel/metrics/data-model/#opentelemetry-protocol-data-model)
+and
+[Prometheus metric formats](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md).
+It has been used to implement Prometheus
+[(pull) exporters for OpenTelemetry SDKs](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/prometheus),
+[OTLP export from Prometheus libraries](https://prometheus.github.io/client_java/otel/otlp/),
+[OTLP ingestion for the Prometheus server](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver),
+and the OpenTelemetry Collector's
+[Prometheus Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver)
+and
+[Prometheus exporters](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusremotewriteexporter).
```

Comment:
The link and the description don't seem to match. This links to "Prometheus Remote Write Exporter" instead of "Prometheus exporters", which one do we intend to have here?

Reply:
By "The OpenTelemetry Collector's ... Prometheus exporters", I was to referencing both the collector's PRW and Prometheus exporters.  I split out PRW and prometheus (pull) exporter to clarify.